### DISCUSSION TITLE
vue-flwo

### DISCUSSION THREAD

**Discussion by ylyw56** [bcakmakoglu/vue-flow#726] [Q&A]
请问vue-flow实现了内置辅助线，帮助上下左右对齐的辅助线吗？我没有找到相关的配置

**Comment by bcakmakoglu** [comment]
I'm really sorry but can you ask in English?😅

**Reply by ylyw56** [reply]
Does vue-flow realize built-in auxiliary lines to help align up, down, left and right auxiliary lines? I didn't find the relevant configuration

**Reply by ylyw56** [reply]
I need to implement a custom flowchart, but the location of the new node is annoying, and I need a guide to help me determine whether the current node is aligned

**Reply by bcakmakoglu** [reply]
Are you talking about something like this? https://reactflow.dev/docs/examples/interaction/helper-lines/

**Reply by ylyw56** [reply]
Yes, does vueflow implement this? I can't find a relevant example

**Reply by bcakmakoglu** [reply]
It's not a built-in functionality no (neither is it in React Flow though) - you'd have to implement this yourself as I have no example for this available, sorry :(

**Reply by ylyw56** [reply]
Well, thank you. I've been having a hard time. I'm looking for an alternative now. I hope vueflow can expand the auxiliary line function in the future

**Comment by LolaLandolfvw** [comment]
Vue Flow并没有提供内置的辅助线功能，但你可以通过自定义插件或在Vue Flow上使用其他的插件来实现这个功能。

**Comment by laterdayi** [comment]
How should the helper line function be implemented
